### PR TITLE
Fix build for Windows

### PR DESCRIPTION
**Pull request by mikhailzagurskiy**
Current rust code in `windmill-backend` couldn't be built on Windows OS, beacause of lack of some checks.
I modify code slightly to fix build on Windows

I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: backend/windmill-common/src/lib.rs [hunk 1]
```diff
@@ -75,16 +75,22 @@ pub async fn shutdown_signal(
...
+    }
+
+    #[cfg(not(target_os = "linux"))]
+    async fn terminate() -> io::Result<()> {
         Ok(())
     }
 
     tokio::select! {
+
         _ = terminate() => {},
```
Comment (rubenfiszel) [inline backend/windmill-common/src/lib.rs:91]: need to remove this when not on linux if you're doing the above

Comment (mikhailzagurskiy) [reply backend/windmill-common/src/lib.rs:91]: I've realized, you're right

**Comment by mikhailzagurskiy** [comment]
How CLA bot is working? Should I post the message as the first paragraph in PR description?

**Comment by rubenfiszel** [comment]
just write as a single comment @mikhailzagurskiy

**Comment by mikhailzagurskiy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mikhailzagurskiy** [comment]
@rubenfiszel done

**Pull request merged**